bundles/org.eclipse.swt/Eclipse SWT/cocoa/org/eclipse/swt/graphics/TextLayout.java

```diff
@@ -38,6 +38,9 @@
  */
 public final class TextLayout extends Resource {
 
+    // Mac features might not handle big integer values (like 0x7fffffff = Integer.MAX_VALUE) since MacOS26.1.
+    	private final static int MAX_VALUE = 10000000;
```

Comment:
Should be `0x...`?

Reply:
`MAX_VALUE_<WHAT?>`.
Can we be more specific here? What we would like to express essentially is "unlimited", so something like

`VALUE_RECT_HUGE_EFFECTIVELY_UNLIMITED`